[PATCH] remove -Werror from CFLAGS

-Werror should only be used during development, not in released SW.

Signed-off-by: Peter Korsgaard <jacmet@sunsite.dk>
---
 Makefile |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: a/Makefile
===================================================================
--- a/Makefile
+++ b/Makefile
@@ -46,7 +46,7 @@
 
 # options
 override CPPFLAGS += -Iinc/
-override CFLAGS += $(OPTIM) -fPIC -Wall -Wextra -Wno-unused-parameter -Wno-unused-result -Werror -Iinc/ $(DEFINES)
+override CFLAGS += $(OPTIM) -fPIC -Wall -Wextra -Wno-unused-parameter -Wno-unused-result -Iinc/ $(DEFINES)
 override LDFLAGS += 
 
 ifeq ("$(OS)", "Darwin")
